src/common/com/intellij/plugins/haxe/HaxeComponentType.java

```diff
@@ -60,12 +61,12 @@ public Icon getCompletionIcon() {
   }, FUNCTION(3) {
     @Override
     public Icon getIcon() {
-      return AllIcons.Nodes.Function;
+      return HaxeIcons.Method;
```

Comment:
was changing from Function to Method intentional here? it looks like you swapped FUNCTION and METHOD